packages/node_modules/@webex/plugin-meetings/test/integration/spec/journey.js

```diff
@@ -410,7 +410,7 @@ skipInNode(describe)('plugin-meetings', () => {
             })
         ])));
 
-      it('alice shares the screen with highFrameRate', () => alice.meeting.getMediaStreams({
+      skipInSafari(it)('alice shares the screen with highFrameRate', () => alice.meeting.getMediaStreams({
```

Comment:
Can you add `TODO` to re-enable when fixes are made for Safari?

Reply:
You want me to do that with all the `skipInSafari`'s?

Reply:
Yes, please